mars/dataframe/expressions/arithmetic/add.py

```diff
@@ -27,11 +28,12 @@ class DataFrameAdd(DataFrameOperand, DataFrameBinOpMixin):
     _axis = AnyField('axis')
     _level = AnyField('level')
     _fill_value = Float64Field('fill_value')
+    _scalar_value = Float64Field('scalar_value')
```

Comment:
I think this field is unnecessary. The `inputs` is enough. We can call `pd.add(*op.inputs)` in execution, what we need do is to check the legality.

Reply:
hi, do you mean `chunk.op.inputs`(before refactor). I found that input only include the dataframe part.

Reply:
OK, that's a problem, thus we can introduce the `_lhs` and `_rhs` fields in tensor, by calling `pd.add(op.lhs, op.rhs)` could look better.